### PR TITLE
spatial extension v1

### PR DESCRIPTION
**Pull request by pfeiferl**
Hey guys, I just finished a new spatial extension for scdrake, especially focusing on data generated by Visium technology. Features added: spatial pseudo-tissue visualization, spatially variable genes selection, and manual annotation using expression signatures (for both single-cell and spatial data). Please, kindly review these sections once you have time. I hope everything will work fine this time. Thank you!

### CODE REVIEW

**Comment by vucinick** [comment]
Hi,

I started the review a week ago and noted several comments regarding semantics. I recommended to Lucie (@pfeiferl) to use the  [lintr](https://lintr.r-lib.org/) package, which will allow me to focus more on the logic of the code. Writing this here to document our in-person discussion.

Thanks! 😃

**Comment by pfeiferl** [comment]
Hi, 

thank you! As suggested, I formatted the code (but used styler as recommended by @gorgitko). I am now waiting for the finished patch by @vucinick, then I will merge the pipelines, and push changes ☺️

File: _drake_single_sample.R
Comment (gorgitko) [inline _drake_single_sample.R:1]: @pfeiferl I think this was added by a mistake.

Comment (gorgitko) [reply _drake_single_sample.R:1]: I've removed the file and amended your commit. Run `git pull --rebase` please.

Comment (pfeiferl) [reply _drake_single_sample.R:1]: Yes, you are right, thank you!

**Review by gorgitko — APPROVED** [review]

**Pull request merged**